iib/web/static/api_v1.yaml

```diff
@@ -967,6 +1006,54 @@ components:
                 This determined what level of protection the addition or removal had.
               type: string
               example: 'prod'
+    RequestCreateEmptyIndex:
+      type: object
+      properties:
+        binary_image:
+          type: string
+          example: 'quay.io/operator-framework/upstream-registry-builder:v1.5.9'
+        from_index:
+          type: string
+          example: 'quay.io/iib-stage/iib:4'
+        labels:
```

Comment:
Check out the `Examples of Dictionary Contents` section here: https://swagger.io/docs/specification/data-models/dictionaries/
We should probably do that too.

Reply:
Can you check if it is alright now?